router/batchrouter/handle_async.go

```diff
@@ -560,17 +590,27 @@ func (brt *Handle) createFakeJob(jobID int64, parameters stdjson.RawMessage) *jo
 	}
 }
 
-func (brt *Handle) getReportMetrics(statusList []*jobsdb.JobStatusT, parametersMap map[int64]stdjson.RawMessage) []*utilTypes.PUReportedMetric {
+type GetReportMetricsParams struct {
+	StatusList    []*jobsdb.JobStatusT
+	ParametersMap map[int64]stdjson.RawMessage
+	JobsList      []*jobsdb.JobT
+}
```

Comment:
Does these needs to be exported? Same for `SetMultipleJobStatusParams` as well?
```suggestion
type getReportMetricsParams struct {
	statusList    []*jobsdb.JobStatusT
	parametersMap map[int64]stdjson.RawMessage
	jobsList      []*jobsdb.JobT
}
```